wp-content/themes/eighshift-task/src/blocks/custom/image-text/components/image-text-editor.js

```diff
@@ -0,0 +1,83 @@
+import classnames from 'classnames';
+
+import { ImageEditor } from '../../../components/image/components/image-editor';
+import { HeadingEditor } from '../../../components/heading/components/heading-editor';
+import { ParagraphEditor } from '../../../components/paragraph/components/paragraph-editor';
+import { ButtonEditor } from '../../../components/button/components/button-editor';
+
+export const ImageTextEditor = (props) => {
+  const {
+    attributes: {
+      includeBtn,
+      blockClass,
+      mediaUrl,
+      mediaPosition,
+      headingContent,
+      headingStyleAlign,
+      headingStyleColor,
+      headingStyleSize,
+      paragraphContent,
+      paragraphStyleAlign,
+      paragraphStyleColor,
+      paragraphStyleSize,
+      btnTitle,
+      btnStyleSize,
+      btnStyleColor,
+      btnStyleSizeWidth,
+    },
+    actions,
+  } = props;
+
+  const blockClasses = classnames([
+    `${blockClass}__position--${mediaPosition}`,
```

Comment:
position  of what?

you are saying that position is left, but of what, image or text?
a better selector would be example: `${blockClass}__image-position--${mediaPosition}`